All project code/index.js

```diff
@@ -196,6 +195,49 @@ app.get("/profile", (req, res) => {
     }});
 });
 
+//using axios Check status for flights that scheduled
+//plug lat and long into 
+//https://airlabs.co/docs/flights
+// database configuration
+
+app.get("/flights", (req, res) => {
+    res.render("pages/flights", {
+        results: [],
+    });
+});
+
+app.post("/flights", (req, res) => {
+    axios({
+        url: "http://api.aviationstack.com/v1/flights",
+        method: 'GET',
+        dataType:'json',
+        params: {
+            "access_key": process.env.flight_api_key,
+            "limit": 40,
+            "flight_status": "scheduled",
+            "arr_icao": req.body.city,
```

Comment:
Even better that this seems to be the only user input we need to take. That data will come from the search bar once the city coordinate API is complete.